server/oauth2.go

```diff
@@ -245,6 +245,8 @@ type idTokenClaims struct {
 	Groups []string `json:"groups,omitempty"`
 
 	Name string `json:"name,omitempty"`
+
+	Login string `json:"dex.coreos.com/login"`
```

Comment:
I was worried that `connector` is also a bit vague if you don't deal with dex's internals. Will think about it.